tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es2017",
+		"lib": [
+			"es6",
+			"es7",
```

Comment:
For consistency with `target`, these should be:

```json
"es2017"
```

You don't need to specify older versions, as `es2017` imports those.